app/components/forms/wizard/wizard-footer.hbs

```diff
@@ -23,12 +23,14 @@
     {{/if}}
     {{#if (eq @event.state 'draft')}}
       <button class="ui blue button right labeled icon small right aligned {{if @loading 'disabled'}}" type="button" {{action @onSaveDraft}}>
-        {{t 'Save as draft'}} <i class="save icon"></i>
+      {{#if this.device.isMobile 'mt-8'}}
+      {{t 'Save as draft'}} <i class="save icon"></i>
```

Comment:
what are you trying to do, what is your logic? I don't think u even know what this code is doing.